sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_client.py

```diff
@@ -114,6 +114,7 @@ def __init__(self, key, credential, **kwargs):
 
         self._local_provider = NoLocalCryptography()
         self._initialized = False
+        self._local_only = kwargs.pop("local_only", False)
```

Comment:
If you don't want this argument to be public, its name should make that clear:
```suggestion
        self._local_only = kwargs.pop("_local_only", False)
```